commons-packet/commons-packet-manager/src/main/java/io/mosip/commons/packet/impl/OnlinePacketCryptoServiceImpl.java

```diff
@@ -298,8 +260,9 @@
     }
 
     @Override
-    public boolean verify(String machineId,byte[] packet, byte[] signature) {
+    public boolean verify(String refId, byte[] packet, byte[] signature) {
        try {
+           String machineId = refId.split("_")[1];
```

Comment:
here we are trying to get the machine id form referenceId, is there a way to get the machine Id without depending on the refId ?

Reply:
Unlike applicationid and additionalInfoReqId  the refid is not something random and its defined that country has to use refid as center_machine id, i have not introduced another variable.